docs/deploying-to-production.md

```diff
@@ -0,0 +1,132 @@
+### Starting the app
+
+Start the server:
+
+`yarn start`
+
+## Deploying to Heroku
+
+* [Forking the repository](#forking-the-repository)
+* [Creating new app](#creating-new-app)
+* [Connect GitHub to Heroku](#connect-github-to-heroku)
+* [Heroku settings](#heroku-settings)
+  * [Updating enviroment variables](#updating-enviroment-variables)
+  * [Domains and certificates](#domains-and-certificates)
+* [Heroku logs](#heroku-logs)
+
+Before creating the app you need three accounts: [Heroku](https://heroku.com/), [Stripe](https://stripe.com/fi) and [MapBox](https://www.mapbox.com/). Creating the accounts is free unless you start heavily using them.
+
+### Forking the repository
+
+When deploying to Heroku you should use a forked repository. This makes it possible to make
+[manual deploys](https://devcenter.heroku.com/articles/github-integration#manual-deploys) after
+changes to environment variables.
+
+If you haven't done this already see [Customization guide](./customization-guide.md#fork-the-repository) for more
+instructions.
+
+### Creating a new app
+
+The easiest way is to click the _"Deploy to Heroku"_ button (check the root folder in your forked
+repository) and fill in the needed information.
+
+After deploying you will find your app from your Heroku [dashboard](https://dashboard.heroku.com/).
+If you click your application you will see Overview and some other tabs.
+
+If you don't want to use the deploy button you can also manually create a new application in Heroku
+dashboard but then you have to manually set all the environment variables. If you prefer using the
+command line you can also use [Heroku CLI](https://devcenter.heroku.com/articles/git).
+
+### Connect GitHub to Heroku
+
+Connecting your GitHub account to Heroku makes it possible to do automatic and manual deploys in the
+dashboard using GitHub branches.
+
+1.  In Heroku dashboard go to _Deploy tab_ and find _Deployment method_ section.
+2.  Click  _Connect to GitHub_ button. After that, you will see _Connect to GitHub_ section.
+3.  Search the repository you have forked to your account and click connect.
+
+![Connect GitHub to Heroku](./assets/deploying-to-production/heroku-connect-git.png)
+
+After this, you can enable automatic deploys from specific GitHub branch or do the manual deploy
+from any branch in the repository.
+
+Read more from Heroku docs:
+
+* [GitHub Integration](https://devcenter.heroku.com/articles/github-integration)
+
+### Heroku settings
+
+In the _Settings tab_ you can manage your application's information, set it to maintenance mode and
+delete the app if needed.
+
+#### Updating enviroment variables
+
+In the _Settings tab_ click _"Reveal Config Vars"_ button to see the applications environment variables.
+Click edit to update them or add new ones if needed.
+
+> **NOTE:** If you change these variables, you need to deploy the app again.
+
+![Heroku settings](./assets/deploying-to-production/heroku-config-vars.png)
+
+#### Domains and certificates
```

Comment:
@sktoiva Do you have anything else that should be written about domain & SSL certs? 
(And read the whole doc)